.dev/src/blocks/product/components/inspector.js

```diff
@@ -65,6 +65,13 @@ const Inspector = ( { posts, media, attributes, setAttributes } ) => {
 					value={ attributes.button_label }
 					onChange={ ( buttonLabel ) => setAttributes( { button_label: buttonLabel } ) }
 				/>
+				<CheckboxControl
+					label={ __( 'Open in New Tab', 'reseller-store' ) }
```

Comment:
**nitpick (non-blocking):** Casing does not match the other description for this.

`Display results in a new tab` vs `Open in New Tab`

Also latter misses the `a` too. Make these as consistent as possible whilst still making sense in context.